spec/index.bs

```diff
@@ -7241,13 +7241,13 @@ dictionary GPURenderPassDepthStencilAttachment {
     required GPUTextureView view;
 
     float depthClearValue = 0;
-    required GPULoadOp depthLoadOp;
-    required GPUStoreOp depthStoreOp;
+    GPULoadOp depthLoadOp;
+    GPUStoreOp depthStoreOp;
```

Comment:
I think we need validation that requires depthLoad/StoreOp if `view` has depth, and ditto for stencil.

Reply:
Done.